python_cdk/src/detectors/python-cdk-redshift-backup-enabled/python-cdk-redshift-backup-enabled_non-compliant.py

```diff
@@ -0,0 +1,24 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-cdk-redshift-backup-enabled@v1.0 defects=1}
+import aws_cdk as cdk
+from aws_cdk.aws_redshift import CfnCluster
+from aws_cdk import  Stack 
+
+class CdkStarterself(cdk.self):
+    def __init__(self, scope: cdk.App, id: str):
+        super(scope, id)
+     
+        # Noncompliant: `automated_snapshot_retention_period` is set to 0
```

Comment:
"# Noncompliant: The CfnCluster instantiation sets the automated_snapshot_retention_period property to 0, disabling automated snapshots retention."